convention.md

```diff
@@ -337,7 +342,7 @@ The format attribute specifies restrictions or options for the given data type.
 | float        | no       | `:`      | `min:max` where min and max are the respective minimum and maximum (inclusive) allowed values, both represented in the format for [float types](#float). Eg. `10.123:15.123`. If the minimum and/or maximum are missing from the format, then they are open-ended, so `0:` allows a value >= 0.|
 | integer      | no       | `:`      | `min:max` where min and max are the respective minimum and maximum (inclusive) allowed values, both represented in the format for [integer types](#integer). Eg. `5:35`. If the minimum and/or maximum are missing from the format, then they are open-ended, so `:10` allows a value <= 10 |
 | enum         | yes      |          | A comma-separated list of non-quoted values. Eg. `value1,value2,value3`. If quotes or whitespace are used adjacent of the '`,`' (comma), then they are part of the value. Individual values can not be an empty string, hence at least 1 value must be specified in the format. |
-| color        | yes      |          | `rgb` or `hsv`. See the [color type](#integer) for the resulting value formats. |
+| color        | yes      |          | `rgb`, `hsv` or `xyz`. See the [color type](#color) for the resulting value formats. |
```

Comment:
This table should also get an entry for `json`

Reply:
Some comments;
 - since this format field goes in the JSON description document, the schema itself does not need to be a sytring, but can be a json object (otherwise you'd get escaped json in a string).
 - if there is no schema provided, then the minimum validation should be that the data is a valid array or object.
 - I think we should refrain from recommending json-schema versions, but just list the challenges and caveats of using them, so folks are warned to do the right-thing™️

Reply:
as for validation, the default `format` for a `json` type should be;
```json
{"anyOf": [{"type": "array"},{"type": "object"}]}
```